Source/CarthageKit/Xcode.swift

```diff
@@ -468,6 +468,30 @@ public enum ProductType: String {
 	}
 }
 
+/// Describes the type of Mach-O files.
+/// See https://developer.apple.com/library/mac/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html#//apple_ref/doc/uid/TP40003931-CH3-SW73.
```

Comment:
Sorry, you should leave it in. I was just joking because it's frustrating how Apple breaks these links all the time. :relaxed: